test/functional/rpc_setban.py

```diff
@@ -60,24 +60,6 @@ def run_test(self):
         assert not self.is_banned(node, tor_addr)
         assert not self.is_banned(node, ip_addr)
 
-        node.setban(tor_addr, "add")
-        assert self.is_banned(node, tor_addr)
-        assert not self.is_banned(node, ip_addr)
-
-        self.log.info("Test the ban list is preserved through restart")
-
-        self.restart_node(1)
-        assert self.is_banned(node, tor_addr)
-        assert not self.is_banned(node, ip_addr)
-
-        node.setban(tor_addr, "remove")
-        assert not self.is_banned(self.nodes[1], tor_addr)
-        assert not self.is_banned(node, ip_addr)
```

Comment:
Same for here.  If this were kept, it would continue to test ban removal for onion addresses.

Reply:
Done.